File: Python/1065.py
a = int(input())
b = int(input())
c = int(input())
d = int(input())
e = int(input())

s=0

for n in [a,b,c,d,e]:
  if n % 2 ==0:
    s = s + 1
print('{} valores pares'.format(s))

File: CSharp/1065.cs
using System; 

class URI {

    static void Main(string[] args) 
    {
        int cont=0;
        int num;

        for(int i=0;i<5;i++)
        {
            num = int.Parse(Console.ReadLine());
            if(num%2==0) cont++;
        }
        Console.WriteLine($"{cont} valores pares");
    }

}